methodoverride/example_polymorphism.py

```diff
@@ -0,0 +1,26 @@
+class India():
```

Comment:
how it is polimorphism if you have not defined the base class? I know it is an implicit way of doing it but defining an interface/abstract class would make it more explicit.